pootle/apps/pootle_misc/checker.py

```diff
@@ -0,0 +1,288 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think @julen had an idea of not using ``pootle_misc`` going forward but moving such things into ``pootle.core.misc`` IIRC.